com.openmusicplayer.OMP.json

```diff
@@ -0,0 +1,257 @@
+{
+
+  "app-id": "com.openmusicplayer.OMP",
+
+  "runtime": "org.gnome.Platform",
+  "runtime-version": "3.30",
+
+  "sdk": "org.gnome.Sdk",
+
+
+  "finish-args":
+  [
+
+    "--socket=x11",
+    "--share=network",
+    "--filesystem=host",
```

Comment:
The Flathub maintainers probably won't like this, especially @barthalion.
Doesn't it support portals?

Reply:
No need to put my nick as an argument, really.

Reply:
Won't like filesystem=host? What would be the most appropriate option?

Reply:
I would say it's okay, because it gives you access to removable media (`/run/media`) and custom locations (e.g. `/data`). What's more, thanks to this you gain read-only access  to the host's `/usr` dir (via `/var/run/host`), which can be useful if you have any sample files in `/usr/share`.
Barthalion, in turn, would like to restrict access to files as much as possible, so he would probably choose the `--filesystem=xdg-music`.
In my opinion, this app should have at least the `--filesystem=home` permissions.

Reply:
I will change it to xdg-music.

Reply:
Still not done;